### DISCUSSION TITLE
Remember credentials at login page

### DISCUSSION THREAD

**Discussion by afatac** [YetiForceCompany/YetiForceCRM#17023] [General]
I changed the following setting to true in config/Security.php and thought that I would see a relevant checkbox on login page.

```
/** Remember user credentials */
public static $LOGIN_PAGE_REMEMBER_CREDENTIALS = true;
```

But nothing is changed at the login page. Did I miss out something?

**Comment by rskrzypczak** [comment]
Not really. This functionality only activates the autocomplete for the field, thanks to which you will see a list with the last logins that you have entered into this field.

**Reply by afatac** [reply]
Nothing happen after I logout and type in the username field again.
